### ISSUE TITLE
Feature request: Add possibility to define table slots

### ISSUE THREAD

**Issue by ndabAP** [vuikit/vuikit#62]
Let's say I would like to add a button to every row of my table. It would be nice to have a slot for this like:

```
<vk-table selectable
  trackBy="id"
  :fields="[{
    name: 'name',
    sortBy: true
  }, {
    name: 'hits',
    sortBy: true,
    headerClass: 'vk-table-width-minimum'
  }, {
    name: 'desc',
    header: 'Description'
  }, {
    name: 'interact' 
  ]"
  :rows="[
    { id: 1, name: 'Item A', hits: 100, desc: 'Description' },
    { id: 2, name: 'Item B', hits: 40, desc: 'Description' },
    { id: 3, name: 'Item C', hits: 700, desc: 'Description' }
  ]"
  @sort="sortOrder = arguments[0]"
  @change="rows = arguments[0]">
  <template slot="interact" scope="item">
    <button>Delete</button>
  </template>
</vk-table>
```

BootstrapVue also uses this technique:

https://bootstrap-vue.github.io/docs/components/table

**Comment by miljan-aleksic** [comment]
@ndabAP, thank you for your feedback.

Current 0.6 version is closed for features addition, but the next 0.7 already has support for slots. It would look something like:

```
<vk-table>
  <vk-table-column-select />
  <vk-table-column-sort header="ID" cell="id" />
  <vk-table-column-sort header="Company" cell="company" />
  <vk-table-column
    header="Website"
    class="uk-text-right"
    cellClass="uk-text-right">
    <template scope="row">
      <a :href="`http://${row.website}`">{{ row.website }}</a>
    </template>
  </vk-table-column>
</vk-table>
```

It's in heavy development, though. No ETA yet.

**Comment by ndabAP** [comment]
Ah very cool. Now I can switch to Vuikit completely.
